index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <title>Hammad's Portfolio </title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+    <div class="app-wrap">
+    <header>
+        <nav class="toolbar">
+            <div class="logo">
+                My Logo
+            
+            <a href="#" class="button">
+              <i class="fa fa-bars"></i>
+            </a> 
+        </div>
+        </nav>
+    </header>
+
+    <main class="page">
+        <section class="intro">
+            <h2 class="name">I'm Hammad</h2>
```

Comment:
- [ ] Please add this title inside `<h1>` tag instead of `<h2>` because it will be better semantically that way.